file-native/src/main/java/org/ballerinalang/stdlib/file/service/DirectoryListenerConstants.java

```diff
@@ -28,7 +28,7 @@
 public class DirectoryListenerConstants {
 
     public static final String MODULE_NAME = "file";
-    public static final String MODULE_VERSION = "0.5.1";
+    public static final String MODULE_VERSION = "0.5.2";
```

Comment:
Why do we need two different variables for module version?

Reply:
Earlier the directory listener was not part of the file module and therefore when merging the two, the files had been kept as it is. This will be addressed with the file module revamp currently in progress.